package.json

```diff
@@ -64,6 +64,13 @@
     "@types/sinon": "~10.0.0",
     "@types/uuid": "~8.3.0",
     "@types/vscode": "~1.52.0",
+    "@typescript-eslint/eslint-plugin": "^4.22.0",
+    "@typescript-eslint/eslint-plugin-tslint": "^4.22.0",
+    "@typescript-eslint/parser": "^4.22.0",
+    "eslint": "^7.25.0",
```

Comment:
I did manually update this to 7.25 since the converter brings in 6.x, but even before that, there were thousands of errors.